tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "resolveJsonModule": true,
     "noEmit": true,
     "jsx": "react",
-    "isolatedModules": true
+    "isolatedModules": true,
+    "noFallthroughCasesInSwitch": true
```

Comment:
Why no fallthrough? Aren't there legitimate use cases for this?

Reply:
Not sure how that got in there whoops